Dockerfile

```diff
@@ -3,12 +3,21 @@ FROM python:3.7-alpine3.9
 ENV PYTHON_UNBUFFERED 1
 ENV PYTHONWARNINGS "ignore:Unverified HTTPS request"
 
-RUN apk add --update git ca-certificates && rm -rf /var/cache/apk/*
+# Create user and homedir
+RUN set -x \
+    && addgroup -g 101 -S indexer \
+    && adduser -S -D -u 101 -h /home/indexer -s /sbin/nologin -G indexer -g indexer indexer
+
+RUN apk add --no-cache --update git ca-certificates
 
 ADD requirements.txt /
+RUN pip install --upgrade pip
 RUN pip install -r /requirements.txt
 
 WORKDIR /app
 COPY indexer.py /app/
 COPY mapping.json /app/
+
+USER indexer
```

Comment:
In order to be able to use this image in Kubernetes without the SecurityContext we should put `USER 101`

Reply:
oh